otter/convergence/planning.py

```diff
@@ -84,7 +84,11 @@ def _remove_from_lb_with_draining(timeout, nodes, now):
 
     changes = [drain_lb_node(node=node) for node in to_drain]
 
-    return removes + changes
+    retry = (
+        [ConvergeLater(reasons=[ErrorReason.String('draining servers')])]
+        if in_drain else [])
```

Comment:
So I've been considering if we should be creating one of these per server. It would mean we can clearly see which server caused which step (if we include a message like `'draining server %s' % (server.id,)`). But maybe that would make the log messages too long. What do you think?

(I was thinking about this since I'm also adding another case of ConvergeLater in another PR).